shared/build.gradle.kts

```diff
@@ -130,6 +130,8 @@ kotlin {
                 api(libs.multiplatform.settings.datastore)
                 api(libs.androidx.datastore)
                 api(libs.androidx.datastore.preferences)
+
+                implementation(libs.apollo.debug.server)
```

Comment:
Anyone knows if it's possible in a MP project to add a dependency for debug builds only (like `debugImplementation` in regular Android builds)?

Reply:
Would the `androidDebug` source set work?

```kotlin
kotlin {
  sourceSets {
    getByName("androidDebug").apply {
      implementation(libs.apollo.debug.server)
    }
  }
}
```

Reply:
Oh my, Gradle lifecycle never ceases to amaze!